src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -161,11 +161,14 @@ public bool PreAuthenticate
             get { return true; }
             set
             {
+                /*
+                TODO:#18104
```

Comment:
The problem is that the PNSE fires every time an HttpWebRequest is sent even with pure defaults for all the settings. The underlying issue is that by default HttpWebRequest.PreAuthenticate is set to false. This value is then set to HttpClientHandler.PreAuthenticate when the SendRequest is called. And that leads to PNSE.
Several months ago HttpWebRequest didn't have the PreAuthenticate property and left the defaults up to the HttpClientHandler, but now it doesn't. And in the meantime it changed the defaults, which is a breaking change for UWP.
Note that this is just a workaround to avoid the breaking change, the correct solution probably looks different (I assume we would change the defaults for HttpWebRequest to match those in Win INet or something along those lines)